testdata/p4_16_samples_outputs/fabric_20190420/fabric-midend.p4

```diff
@@ -430,7 +432,9 @@ control FabricIngress(inout parsed_headers_t hdr, inout fabric_metadata_t fabric
     @name("FabricIngress.forwarding.mpls_counter") direct_counter(CounterType.packets_and_bytes) forwarding_mpls_counter;
     @name("FabricIngress.forwarding.pop_mpls_and_next") action forwarding_pop_mpls_and_next_0(@name("next_id") bit<32> next_id_6) {
         fabric_metadata._mpls_label5 = 20w0;
-        fabric_metadata._next_id10 = next_id_6;
+        @hidden {
```

Comment:
@ChrisDodd Sorry I haven't spent a lot of time looking through this test program, but is it clear where the `@hidden` annotation in this expected output file is coming from?

I guess it is from an action that was in-lined into another action body?

Reply:
Actually, it is the `DoRemoveExits` pass that was previously removing the annotation.

Reply:
Thanks for the detailed explanation!  Makes sense that your fixes would lead to these annotations appearing in the midend output files, then.